app/controllers/curation_concerns/permissions_controller.rb

```diff
@@ -0,0 +1,13 @@
+class CurationConcerns::PermissionsController < ApplicationController
+  include CurationConcerns::PermissionsControllerBehavior
+
+  def confirm_access
+  end
```

Comment:
@awead @jcoyne added a comment.  Not sure I ❤️ the new review workflow becuase it is not showing the new code.